src/Clapi/Tree.hs

```diff
@@ -195,3 +195,17 @@ roseTreeNode t = case t of
 
 treeLookupNode :: Path -> RoseTree a -> Maybe (RoseTreeNode a)
 treeLookupNode p = fmap roseTreeNode . treeLookup p
+
+data RoseTreeNodeType
+  = RtntEmpty
+  | RtntContainer
+  | RtntConstData
+  | RtntDataSeries
+  deriving (Enum, Show)
```

Comment:
Do we use the `Enum` instance? It should probably have `Eq`, `Ord` and `Bounded` if we're adding "things it is" derivations, or it should probably not have any of them...